build.gradle

```diff
@@ -18,6 +18,13 @@ buildscript {
     }
 }
 
+plugins {
+    id 'application'
+    id 'org.openjfx.javafxplugin' version '0.0.8'
+}
+
+mainClassName = '...'
```

Comment:
I don't think this is needed if it's empty....

Reply:
that mainClassName was fixing the client [build]